templates/common/openstack/units/openstack-hostname.service.yaml

```diff
@@ -14,19 +14,9 @@ contents: |
   After=afterburn.service
 
   [Service]
-  # Would prefer to do Restart=on-failure instead of this bash retry loop, but
-  # the version of systemd we have right now doesn't support it. It should be
-  # available in systemd v244 and higher.
-  ExecStart=/bin/bash -c " \
-  until \
-  /usr/local/bin/openstack-afterburn-hostname; \
-  do \
-  sleep 10; \
-  done"
   Type=oneshot
-  TimeoutSec=600
-  # Show as active after exit since we change state
-  RemainAfterExit=true
+  RemainAfterExit=yes
+  ExecStart=/usr/local/bin/mco-hostname --openstack
```

Comment:
Is it safe to drop the retry loop?

It was added in https://github.com/openshift/machine-config-operator/commit/3bc24f6f29c5b7f0b08f5610e5a3aaf90002f18f, and later modified in https://github.com/openshift/machine-config-operator/commit/669db1366679834890dff365a6a5b32436de487b.

My impression is that we still want it.

Reply:
you're right, I can re-add it.